de.zray.Facetracker.json

```
@@ -0,0 +1,62 @@
+{
+  "app-id": "de.zray.Facetracker",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "facetracker-gui",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=all",
+    "--socket=wayland"
+  ],
+  "modules": [
+    {
+      "name": "facetracker-addional-files",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D de.zray.Facetracker.desktop /app/share/applications/",
```

```suggestion
        "install -Dm0644 de.zray.Facetracker.desktop /app/share/applications/${FLATPAK_ID}.desktop",
```

Or use `-t` this will fail otherwise.